.circleci/config.yml

```diff
@@ -0,0 +1,240 @@
+version: 2.1
+
+parameters:
+  ci_builder_image:
+    type: string
+    # depends with rvsol/lib/optimism submodule version
+    default: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.46.1
+
+workflows:
+  main:
+    jobs:
+      - pnpm-monorepo
+      - go-mod-download-monorepo
+      - go-mod-download-asterisc
+      - op-program-riscv:
+          requires: ["go-mod-download-monorepo"]
+      - asterisc-prestate:
+          requires: ["go-mod-download-asterisc", "op-program-riscv", "pnpm-monorepo"]
+
+jobs:
+  pnpm-monorepo:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run:
+          name: Fetch submodules for asterisc
+          command: git submodule update --init --recursive
+      - run:
+          name: Check L1 geth version
+          command: ./ops/scripts/geth-version-checker.sh || (echo "geth version is wrong, update ci-builder"; false)
+          working_directory: rvsol/lib/optimism
+      - run:
+          name: git submodules for monorepo
+          command: make submodules
```

Comment:
Do we need this even if we run `git submodule update` with `--recursive` flag in the prior step?

Reply:
No we do not. Removed